### ISSUE TITLE
Runtime out of memory error 

### ISSUE THREAD

**Issue by Shri-Humrudha** [microsoft/Azure-Kinect-Sensor-SDK#576]
Bug description
Cloud points are not viewed in Body tracking viewer

1. Go to body tracking viewer
2. Infra image appears
3. Run time error is printed in the command window
C:\repos\onnxruntime\onnxruntime\core\providers\cuda\cuda_call.cc:97 onnxruntime:cudacall c:\repose\onnxruntime\onnxruntime\core\providers\cuda\cuda_call.cc:91 onnxruntime::cudacall CUDS failure 2: out of memory ; GPU=9; hostname=<machinename> ; expr=cudamalloc((void**)&p, size);
4. No cloud points of the skeleton

Screenshot:
![image](https://user-images.githubusercontent.com/52764236/62383199-24003200-b504-11e9-878e-655ae288daed.png)

Desktop 
 - OS with Version: Windows 10 Enterprise 1903 18362.239
 - SDK Version:  0.9.0
 - Firmware version:  
 == Azure Kinect DK Firmware Tool ==
Device Serial Number: 000212292212
Current Firmware Versions:
  RGB camera firmware:      1.6.98
  Depth camera firmware:    1.6.70
  Depth config file:        6109.7
  Audio firmware:           1.6.14
  Build Config:             Production
  Certificate Type:         Microsoft

**Comment by cdedmonds** [comment]
@Shri-Humrudha it sounds like your GPU does not have enough free memory to run body tracking. Do you have other applications open on your machine?

**Comment by YZR821** [comment]
Hi, I got some problems like this too! I wanna to run my programs for 4 times to get joints information from four different .mvk files, however when processing the fourth video, the cmd line gave me an error information like yours and I have no idea how to solve it. If you found a solution and can share it with me, I'll be appreciate that!

**Comment by cdedmonds** [comment]
@YZR821 and @Shri-Humrudha it sounds like your GPUs are running out of available GPU memory for body tracking. This could be due to opening multiple trackers or other programs that are running.

**Comment by Shri-Humrudha** [comment]
I have only one tracker application opened. It was never working for me
though.

**Comment by cdedmonds** [comment]
Can you please comment on what GPU you are using?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by YZR821** [comment]
@cdedmonds First, let me explain my project to you: I run a piece of code for four time, and each time I create one tracker to get the joints information from a .mkv file. After finishing one .mkv file, I shutdown and destroyed the tracker, then I open another .mkv file and do the same things. However, I noticed that my  GPU memory didn't release after I shutdown and destroy the tracker, and I didn't find any other functions in the SDK to solve this problem. I'm not sure whether I explained my situation clearly, but thanks for your help!
